src/components/Sidebar/Sidebar.js

```diff
@@ -11,7 +11,7 @@ function Sidebar() {
 
   const gardenData = useMemo(
     () =>
-      gardens.slice(0, 10).map(garden => {
+      gardens.slice(0, 100).map(garden => {
```

Comment:
Following previous comment (we´ll need to add the `connectedGarden` as dependency of `useMemo`): 

Also what´s the rationale for increasing the sliced size?

```suggestion
      gardens
        .filter(garden => garden.address === connectedGarden.address)
        .slice(0, 100)
        .map(garden => {
```